src/t2iapi/metric/service.proto

```diff
@@ -35,10 +36,12 @@ service MetricService {
         returns (BasicResponse);
 
     /*
-    Change the value for the metric with the provided handle and confirm the value using the device ui.
-     */
-    rpc SetUserConfirmableValue (BasicHandleRequest)
-        returns (BasicResponse);
+    For the metric with the given handle do the following at once:
```

Comment:
maybe write "simultaneously" instead of "at once"

Reply:
done